Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN locale-gen fr_FR \
-    && locale-gen en_US.UTF-8 \
-    && dpkg-reconfigure locales \
-    && update-locale LANG=en_US.UTF-8 \
-    && update-locale LC_ALL=en_US.UTF-8 \
-    && ln -s /usr/share/i18n/SUPPORTED /var/lib/locales/supported.d/all \
-    && locale-gen \
-    && echo 'LANG="en_US.UTF-8"' > /etc/default/locale
-
-ENV PYTHONIOENCODING="utf-8" \
-    LANG="C.UTF-8" \
```

Comment:
On Thu, Apr 14, 2016 at 5:44 PM, Tulio Ruiz <notifications@github.com>
wrote:

> @LuisAlejandro <https://github.com/LuisAlejandro> We had some issues with
> the locales, Id like to test that with the new changes, but I can't recall
> what was about. @moylop260 <https://github.com/moylop260> do you remember
> anything about this?
>
>
I didnt know either... the encoding must be fixed on global environment as
Luis mentioned...

Whet may be we need to do is ensure all locales are installed and
available, because if not bad things happen but never in the python
environment.


-- 
Nhomar Hernandez
CEO Vauxoo.
Site: http://vauxoo.com
Twitter: @nhomar
Blog: http://nhomar.com
Github User: https://github.com/nhomar
Odoo Gold Partner
Skype: nhomar00 (Envia mail previo no lo superviso siempre).
HangOut: nhomar@vauxoo.com
Móvil Venezuela:
+58 4144110269
Móvil México:
+52 1 4773933942